esdb-client/src/main/java/com/opencqrs/esdb/client/tracing/NoTracingEventEnricher.java

```diff
@@ -0,0 +1,19 @@
+/* Copyright (C) 2025 OpenCQRS and contributors */
+package com.opencqrs.esdb.client.tracing;
+
+import com.opencqrs.esdb.client.EventCandidate;
+
+/** A trivial implementation of {@link TracingEventEnricher} which does not add any tracing data */
+public class NoTracingEventEnricher implements TracingEventEnricher {
+
+    /**
+     * Trivial default implementation of the method which simply returns the given {@link EventCandidate}
```

Comment:
better: Identity method returning the original EventCandidate without altering or enriching any tracing headers.